Update NDless-SDK Docker images

Ndless' Docker images were last updated 2 years ago.
Therefore, my CI which pulls ndless-sdk directly from docker hub crashed with the error message:
```
genzehn --input MyProject.elf --output MyProject.tns.zehn --name "MyProject" --uses-lcd-blit yes
terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::program_options::unknown_option> >'
  what():  unrecognised option '--uses-lcd-blit'
Aborted (core dumped)
make: *** [MyProject.tns] Error 134
```
I'd kindly ask someone who's got access to the `ndless` Docker-Hub account to push a fresh build.

I had to [change Docker.arm-gcc a bit](https://github.com/FrostyPenguin/Ndless/commit/157604f36965bb9201ea0072c23a01c41d38eb38). Now the docker-images are compiling, but a new problem occurred when trying to compile Ndless-programs from inside the docker image (as my CI does):
genzehn crashes with the following error message:
```
nspire-gcc -Wall -W -marm -Os -c main.c -o main.o
mkdir -p .
nspire-ld main.o -o test.elf 
genzehn --input test.elf --output test.tns.zehn --name "test"
genzehn: error while loading shared libraries: libboost_program_options.so.1.65.1: cannot open shared object file: No such file or directory
make: *** [test.tns] Error 127
```
I compiled ndless with boost 1.65.1 on my Arch Linux (the Ubuntu docker image serves 1.54.0). However, this shouldn't be a problem because I suggest toolchain and SDK are compiled in the Docker image.

--- a/ndless-sdk/Dockerfile.arm-gcc
+++ b/ndless-sdk/Dockerfile.arm-gcc
@@ -6,7 +6,7 @@ FROM ndless/gcc
 RUN apt-get update
 
 # install the dependencies
-RUN apt-get install -y libgmp-dev libmpfr-dev libmpc-dev zlib1g-dev libtinfo-dev python2.7-dev libboost-program-options-dev
+RUN apt-get install -y libgmp-dev libmpfr-dev libmpc-dev zlib1g-dev libtinfo-dev python2.7-dev libboost-program-options-dev xz-utils
 
 RUN useradd -m -d /home/ndless -p ndless ndless && chsh -s /bin/bash ndless
 
@@ -16,5 +16,4 @@ ADD . /ndless-sdk
 RUN chown -R ndless:ndless /ndless-sdk && cd /ndless-sdk/toolchain && su ndless -c "./build_toolchain.sh && rm -rf download build-binutils build binutils-* gcc-* gdb-* newlib-* "
 
 ENV PATH /ndless-sdk/toolchain/install/bin:$PATH
-
 

--- a/ndless-sdk/Dockerfile.ndless-sdk
+++ b/ndless-sdk/Dockerfile.ndless-sdk
@@ -7,3 +7,6 @@ RUN chown -R ndless:ndless /ndless-sdk
 
 ENV PATH /ndless-sdk/bin:$PATH
 
+# (Re-)build the SDK
+RUN cd /ndless-sdk && make clean && make
+

